src/plugins/validate-response/checkers/index.ts

```diff
@@ -59,10 +59,21 @@ export const getResponseValue = (
   return 0
 }
 
+/**
+ * responseChecker checks the response against notif/alert triggers
+ * @param {obj} alert contains our alerts to look for
+ * @param {obj} res is the probe result
+ * @returns {boolean} flag true if we have something to alert/notify
+ */
 const responseChecker = (
   alert: ProbeAlert,
   res: AxiosResponseWithExtraData
 ): boolean => {
+  // if status is 599 : timeout or uri is not found (0), worth reporting so return true
```

Comment:
If the timeout came from internet connection, will this report causing another problem?

Reply:
as the test above, the time out worked. thanks!